helpers/posterHelper.js

```diff
@@ -1,16 +1,24 @@
-import { MongoClient, ObjectId} from "mongodb";
+import { MongoClient, ObjectId } from "mongodb";
+
+export async function getAllMoviePosters(numberofposters) {
+    //If there is no input the function will return all posters
```

Comment:
Good to add function to limit how many posters that will show on start page.
Should it also be sorted so that we show the latest movies?

Reply:
> Good to add function to limit how many posters that will show on start page. Should it also be sorted so that we show the latest movies first?

Good idea Mats, I will fix it right away :)